setup.py

```diff
@@ -40,6 +40,7 @@
             'scrapbook~=0.5.0',
             'jupyter',
             'graphviz',
+            'pytest-socket>=0.2',  # 0.2 adds the socket_disabled fixture, cf. https://github.com/scikit-hep/pyhf/pull/1917#discussion_r951943084
```

Comment:
```suggestion
            'pytest-socket>=0.2',  # c.f. PR #1917
```

We can just add this information RE: the API in the commit message